datadog_checks_dev/datadog_checks/dev/tooling/commands/service_checks.py

```diff
@@ -0,0 +1,147 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import json
+from collections import OrderedDict
+
+import click
+from six import string_types
+
+from .utils import CONTEXT_SETTINGS, abort, echo_failure, echo_info
+from ..constants import get_root
+from ...compat import JSONDecodeError
+from ...utils import file_exists, read_file
+
+REQUIRED_ATTRIBUTES = {
+    'agent_version',
+    'check',
+    'description',
+    'groups',
+    'integration',
+    'name',
+    'statuses',
+}
+
+
+def parse_version_parts(version):
+    return (
```

Comment:
I think this would be more readable
```
if not isinstance(version, string_types):
    return []

return [int(v) for v in version.split('.') if v.isdigit()]
```